frontend/src/components/ExpanderButton.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faAngleUp,
+  faAngleDown,
+} from '@fortawesome/free-solid-svg-icons';
+import colors from '../colors';
+
+export default function ExpanderButton({
+  closeOrOpen,
+  count,
+  expanded,
+  number,
+  type,
+  ariaLabel,
+}) {
+  if (count < 1) {
+    return null;
+  }
+
+  return (
+    <button
+      type="button"
+      className={`usa-button--outline usa-button text-no-underline text-middle tta-smarthub--goal-row-${type}s tta-smarthub--goal-row-${type}s-enabled`}

```

Comment:
probably want to change the name of these classes now if we are using them in more than the goal row

Reply:
yes